src/OpenTelemetry.Contrib.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog - OpenTelemetry.Contrib.Instrumentation.AWS
 
-## Unreleased
```

Comment:
By default we keep empty, unreleased section. Could you please bring it back?

Reply:
No problem. Added back.